SwiftCompilerSources/Sources/SIL/ConstExprState.swift

```diff
@@ -0,0 +1,32 @@
+//===--- ConstExprState.swift - -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import AST
+import SILBridging
+
+public struct ConstExprFunctionState {
```

Comment:
Should we call this `ConstExpressionEvaluator`?
I don't know what `ConstExprFunctionState` refers to.